src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import classNames from 'classnames';
+import dropdownStyles from './styles';
+
+interface DropdownOption {
+  id: number,
+  value: string
+}
+
+interface DropdownProps {
+  name: string,
+  options: DropdownOption[],
+  title: string,
+  className?: string,
+}
+
+const renderOptions = (options: DropdownOption[]) => options.map((opt) => (
```

Comment:
I find anonymous functions really messy, hard to debug and not a good practice at all. Maybe we can leave it since it's just a return of a component, but let avoid them specially when they invoke new functions.